### PR TITLE
Patron2020 324 additional api info

### PR DESCRIPTION
**Pull request by kuba8686**

### CODE REVIEW

File: gateway/dashboard.h [hunk 1]
```diff
@@ -210,12 +210,22 @@ class Dashboard
...
+                hvac_status[i].cooling = request["cooling"].get<bool>();
+                sensorExist = 1;
+                return 0;
+            }
+        
+            if (sensorExist == 0) 
+            {
+                add_smoke_sensor(request);
+                return 0;
+            }
```
Comment (szymoneiro) [inline gateway/dashboard.h:225]: Why is add_smoke_sensor function related to HVACStatus? Maybe I get something wrong

File: gateway/dashboard.h [hunk 2]
```diff
@@ -10,7 +10,7 @@ class Dashboard
     std::vector<WindowBlind> window_blinds;
     std::vector<RFIDSensor> rfid_sensors;
     std::vector<SmokeSensor> smoke_sensors;
-    HVACStatus hvac_status;
+    std::vector<HVACStatus> hvac_status;
```
Comment (SzymonFed) [inline gateway/dashboard.h:13]: Why put status in vector if we have only one HVAC?

File: gateway/jsons/dashboard.json [hunk 3]
```diff
@@ -58,15 +58,20 @@
             "isSmokeDetected": false
         }
     ],
-    "HVACStatus": {
+    "HVACStatus": [
+        {
+        "id": 91,
```
Comment (SzymonFed) [inline gateway/jsons/dashboard.json:63]: I can' see why we need an id for HVAC since we have one only.

**Comment by kuba8686** [comment]
@SzymonFed adding id field to hvac status was a part of ticket no 324. I agree it doesnt have to be an object but i will leave it at a moment, because JS and Android group implemented schema compatibile to this.

**Pull request merged**